packages/firestore/lib/modular/index.d.ts

```diff
@@ -701,7 +722,7 @@ export function loadBundle(
  * @param name - The name of the query.
  * @returns A named Query.
  */
-export function namedQuery(firestore: Firestore, name: string): Query<DocumentData>;
+export function namedQuery(firestore: Firestore, name: string): Promise<Query | null>;
```

Comment:
Just double checking, @mikehardy,  in the firebase-js-sdk documentation, it returns a Promise<Query | null>. But we currently return a `Query`. Should we match the firebase-js-sdk in this instance? (I assume yes!).